Submitted By: Jim Gifford <jim@linuxfromscratch.org>
Date: 2006-07-10
Initial Package Version: 2.4
Origin: Various
Upstream Status: Unknown
Description: Fixes build if host distro is using mawk.
	     Distro's known to have the issue Debina
	     and Ubuntu

--- glibc-2.4/scripts/gen-sorted.awk.orig	2006-07-10 23:19:03.474223558 -0700
+++ glibc-2.4/scripts/gen-sorted.awk	2006-07-10 23:21:04.175110403 -0700
@@ -16,7 +16,7 @@
 {
   subdir = type = FILENAME;
   sub(/^.*\//, "", type);
-  sub(/\/[^/]+$/, "", subdir);
+  sub(/\/[^\/]+$/, "", subdir);
   sub(/^.*\//, "", subdir);
   thisdir = "";
 }
